osp/core/session/transport/transport_utils.py

```diff
@@ -256,10 +259,10 @@ def deserialize(json_obj, session, buffer_context, _force=False):
         return [deserialize(x, session, buffer_context, _force=_force)
                 for x in json_obj]
     if isinstance(json_obj, dict) \
-            and set(["UID"]) == set(json_obj.keys()):
-        return convert_to(json_obj["UID"], "UID")
+            and {"UID"} == set(json_obj.keys()):
```

Comment:
True. I simply did not pay attention to this (just replaced the set call by a literal because the IDE suggests so) If you notice the same can be said about the old line.